frontend/pages/equipment/[equipmentID].vue

```diff
@@ -142,7 +121,7 @@ const metricTabs = computed(() => [
   },
   {
     label: "Quality",
-    color: "yellow",
+    color: "indigo",
```

Comment:
:O

Reply:
The yellow was difficult to view on the light background, and going darker wasn't an option because of machine state colors